variants/dfrobot_romeo_esp32s3/pins_arduino.h

```diff
@@ -0,0 +1,82 @@
+#ifndef Pins_Arduino_h
+#define Pins_Arduino_h
+
+#include <stdint.h>
+
+
+
+#define EXTERNAL_NUM_INTERRUPTS 46
+#define NUM_DIGITAL_PINS        48
+#define NUM_ANALOG_INPUTS       20
+
+#define analogInputToDigitalPin(p)  (((p)<20)?(analogChannelToDigitalPin(p)):-1)
+#define digitalPinToInterrupt(p)    (((p)<48)?(p):-1)
+#define digitalPinHasPWM(p)         (p < 46)
+
+
+static const uint8_t TX = 43;
+static const uint8_t RX = 44;
+
+static const uint8_t SDA = 1;
+static const uint8_t SCL = 2;
+
+
+static const uint8_t MOSI  = 15;
+static const uint8_t MISO  = 16;
+static const uint8_t SCK   = 17;
```

Comment:
You also need to add definition for `SS` here

Reply:
Already added and committed